src/modules/users/crons/profile-populator.cron.ts

```
@@ -0,0 +1,341 @@
+      case 'Female':
+        return 'female';
+      default:
+        return null;
+    }
+  }
+
+  private handleClassField(vc: any, pathValue: any) {
+    let value = this.getValue(vc, pathValue);
+    if (!value) return null;
+    value = this.romanToInt(value);
+    if (isNaN(value)) return null;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)